pdg/src/info.rs

```diff
@@ -1,31 +1,83 @@
 use crate::Graphs;
+use crate::graph::{Graph,NodeId,Node,NodeKind};
 use std::fmt::{self, Debug, Display, Formatter};
+use std::collections::HashMap;
 
 /// Information generated from the PDG proper that is queried by static analysis.
 ///
-/// Eventually this will include information about what kinds of [`Node`]s the [`Node`] is an ancestor of,
-/// as well as its ability to be used as a `&mut`.
+/// Includes information about what kinds of [`Node`]s the [`Node`] flows to,
+/// and eventually will also include its ability to be used as a `&mut`.
 ///
 /// [`Node`]: crate::graph::Node
-#[derive(Hash, Clone, PartialEq, Eq, Debug)]
-pub struct NodeInfo {}
+#[derive(Hash, Clone, PartialEq, Debug)]
+pub struct NodeInfo {
+    flows_to: Flows,
+}
+
+/// Contains information about what kinds of [`Node`]s the [`Node`] flows to.
```

Comment:
I think the definition of "flows" needs to be explained here in this documentation